Jib native build failures when setting custom 'quarkus.jib.base-native-image' values

### Describe the bug

Building a native image with the container-jib extension throws a `com.google.cloud.tools.jib.api.RegistryAuthenticationFailedException` when the value of the `quarkus.jib.base-native-image` property is set to anything OTHER than images hosted on `registry.access.redhat.com`.

I recently discovered this when attempting to use docker-hub distributed images of `ubi8-minimal` (for various reasons), i.e. instead of setting `quarkus.jib.base-native-image=registry.access.redhat.com/ubi8/ubi-minimal:8.4` (which does work) I set the property to `quarkus.jib.base-native-image=redhat/ubi8-minimal:8.4`

This also may be related to #8498

***NOTE:** This failure does not occur when building in JVM mode. I can freely use `docker.io` images when building in JVM mode.*

### Expected behavior

Build a native executable image using docker hub distributed images as the native image's base image.

### Actual behavior

Upon executing the following (CI pipeline replaces all env vars with proper values):
```
mvn clean package -Pnative -Dquarkus.container-image.build=true -Dquarkus.container-image.tag=$VERSION-native  -Dquarkus.container-image.username=$USER_CREDENTIALS_USR -Dquarkus.container-image.password=$USER_CREDENTIALS_PSW -Dquarkus.container-image.registry=$REGISTRY_URL -Dquarkus.container-image.push=true -Dquarkus.jib.base-native-image=redhat/ubi8-minimal:8.4
```
I get the following stack trace (note that the actual GraalVM compilation is successful):
```
[WARNING] [io.quarkus.container.image.jib.deployment.JibProcessor] Base image 'redhat/ubi8' does not use a specific image digest - build may not be reproducible
[INFO] [io.quarkus.container.image.jib.deployment.JibProcessor] The base image requires auth. Trying again for redhat/ubi8...
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  03:52 min
[INFO] Finished at: 2021-08-27T20:22:59Z
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal io.quarkus:quarkus-maven-plugin:2.1.4.Final:build (default) on project Project: Failed to build quarkus application: io.quarkus.builder.BuildException: Build failure: Build failed due to errors
[ERROR] 	[error]: Build step io.quarkus.container.image.jib.deployment.JibProcessor#buildFromNative threw an exception: java.lang.RuntimeException: Unable to create container image
[ERROR] 	at io.quarkus.container.image.jib.deployment.JibProcessor.containerize(JibProcessor.java:194)
[ERROR] 	at io.quarkus.container.image.jib.deployment.JibProcessor.buildFromNative(JibProcessor.java:169)
[ERROR] 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
[ERROR] 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
[ERROR] 	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
[ERROR] 	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
[ERROR] 	at io.quarkus.deployment.ExtensionLoader$2.execute(ExtensionLoader.java:820)
[ERROR] 	at io.quarkus.builder.BuildContext.run(BuildContext.java:277)
[ERROR] 	at org.jboss.threads.ContextHandler$1.runWith(ContextHandler.java:18)
[ERROR] 	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2449)
[ERROR] 	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1478)
[ERROR] 	at java.base/java.lang.Thread.run(Thread.java:829)
[ERROR] 	at org.jboss.threads.JBossThread.run(JBossThread.java:501)
[ERROR] Caused by: com.google.cloud.tools.jib.api.RegistryAuthenticationFailedException: Failed to authenticate with registry registry-1.docker.io/redhat/ubi8 because: 401 Unauthorized
[ERROR] GET https://auth.docker.io/token?service=registry.docker.io&scope=repository:redhat/ubi8:pull
[ERROR] {"details":"incorrect username or password"}
[ERROR] 
[ERROR] 	at com.google.cloud.tools.jib.registry.RegistryAuthenticator.authenticate(RegistryAuthenticator.java:306)
[ERROR] 	at com.google.cloud.tools.jib.registry.RegistryAuthenticator.authenticate(RegistryAuthenticator.java:258)
[ERROR] 	at com.google.cloud.tools.jib.registry.RegistryAuthenticator.authenticatePull(RegistryAuthenticator.java:177)
[ERROR] 	at com.google.cloud.tools.jib.registry.RegistryClient.doBearerAuth(RegistryClient.java:336)
[ERROR] 	at com.google.cloud.tools.jib.registry.RegistryClient.authPullByWwwAuthenticate(RegistryClient.java:395)
[ERROR] 	at com.google.cloud.tools.jib.builder.steps.PullBaseImageStep.call(PullBaseImageStep.java:168)
[ERROR] 	at com.google.cloud.tools.jib.builder.steps.PullBaseImageStep.call(PullBaseImageStep.java:67)
[ERROR] 	at com.google.common.util.concurrent.TrustedListenableFutureTask$TrustedFutureInterruptibleTask.runInterruptibly(TrustedListenableFutureTask.java:125)
[ERROR] 	at com.google.common.util.concurrent.InterruptibleTask.run(InterruptibleTask.java:69)
[ERROR] 	at com.google.common.util.concurrent.TrustedListenableFutureTask.run(TrustedListenableFutureTask.java:78)
[ERROR] 	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
[ERROR] 	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
[ERROR] 	at java.base/java.lang.Thread.run(Thread.java:829)
[ERROR] Caused by: com.google.cloud.tools.jib.http.ResponseException: 401 Unauthorized
[ERROR] GET https://auth.docker.io/token?service=registry.docker.io&scope=repository:redhat/ubi8:pull
[ERROR] {"details":"incorrect username or password"}
[ERROR] 
[ERROR] 	at com.google.cloud.tools.jib.http.FailoverHttpClient.call(FailoverHttpClient.java:329)
[ERROR] 	at com.google.cloud.tools.jib.http.FailoverHttpClient.call(FailoverHttpClient.java:250)
[ERROR] 	at com.google.cloud.tools.jib.registry.RegistryAuthenticator.authenticate(RegistryAuthenticator.java:284)
[ERROR] 	... 12 more
[ERROR] Caused by: com.google.api.client.http.HttpResponseException: 401 Unauthorized
[ERROR] GET https://auth.docker.io/token?service=registry.docker.io&scope=repository:redhat/ubi8:pull
[ERROR] {"details":"incorrect username or password"}
[ERROR] 
[ERROR] 	at com.google.api.client.http.HttpRequest.execute(HttpRequest.java:1113)
[ERROR] 	at com.google.cloud.tools.jib.http.FailoverHttpClient.call(FailoverHttpClient.java:323)
[ERROR] 	... 14 more
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoExecutionException
```

Also note that when executing the same command but using `-Dquarkus.jib.base-native-image=registry.access.redhat.com/ubi8/ubi-minimal:8.4` it functions properly:
```
[INFO] [io.quarkus.container.image.jib.deployment.JibProcessor] Starting container image build
[WARNING] [io.quarkus.container.image.jib.deployment.JibProcessor] Base image 'registry.access.redhat.com/ubi8/ubi-minimal:8.4-208' does not use a specific image digest - build may not be reproducible
[INFO] [io.quarkus.container.image.jib.deployment.JibProcessor] Using base image with digest: sha256:31ccb79b1b2c2d6eff1bee0db23d5b8ab598eafd6238417d9813f1346f717c11
[INFO] [io.quarkus.container.image.jib.deployment.JibProcessor] Container entrypoint set to [./application]
[INFO] [io.quarkus.container.image.jib.deployment.JibProcessor] Pushed container image org/group/project:0.1.1-SNAPSHOT-native (sha256:ccc55b94e8fce24ad92732c134a5476d9106818314c622191ba162d6fdb37ee6)

[INFO] [io.quarkus.deployment.QuarkusAugmentor] Quarkus augmentation completed in 332215ms
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  05:49 min
[INFO] Finished at: 2021-08-27T14:55:49Z
[INFO] ------------------------------------------------------------------------
```

### How to Reproduce?

1. Set `quarkus.jib.base-native-image=redhat/ubi8-minimal:8.4`
2. `mvn clean package -Pnative -Dquarkus.container-image.build=true -Dquarkus.container-image.tag=$VERSION-native  -Dquarkus.container-image.username=$USER_CREDENTIALS_USR -Dquarkus.container-image.password=$USER_CREDENTIALS_PSW -Dquarkus.container-image.registry=$REGISTRY_URL -Dquarkus.container-image.push=true -Dquarkus.jib.base-native-image=redhat/ubi8-minimal:8.4`

#### Configuration
quarkus.application.name=project
quarkus.banner.enabled = false
quarkus.container-image.group=org/group
quarkus.native.native-image-xmx=4096m
quarkus.native.additional-build-args=--initialize-at-run-time=org.apache.activemq.util.IdGenerator
quarkus.camel.bootstrap.enabled=false
quarkus.camel.main.enabled=false
quarkus.camel.routes-discovery.enabled=false
quarkus.camel.runtime-catalog.components=false
quarkus.camel.runtime-catalog.languages=false
quarkus.camel.runtime-catalog.dataformats=false
quarkus.camel.runtime-catalog.models=false
quarkus.jackson.fail-on-unknown-properties=false
quarkus.elasticsearch.health.enabled=false
quarkus.elasticsearch.hosts=${ES_HOSTS}
quarkus.elasticsearch.protocol=http
quarkus.log.level=INFO
quarkus.log.console.format=%d{HH:mm:ss} %-5p [%c{2.}] (%t) %s%e%n
quarkus.log.console.color=false
quarkus.log.category."org.apache.activemq".level=INFO
quarkus.log.category."org.apache.camel.component".level=INFO
quarkus.log.category."org.apache.camel.impl".level=INFO
quarkus.log.category."or.ap.ht".level=INFO
quarkus.log.category."org.elasticsearch".level=INFO

### Output of `uname -a` or `ver`

Linux e6d5e3d1a507 3.10.0-1127.el7.x86_64 #1 SMP Tue Mar 31 23:36:51 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux

### Output of `java -version`

openjdk version "11.0.12" 2021-07-20 OpenJDK Runtime Environment GraalVM CE 21.2.0 (build 11.0.12+6-jvmci-21.2-b08) OpenJDK 64-Bit Server VM GraalVM CE 21.2.0 (build 11.0.12+6-jvmci-21.2-b08, mixed mode, sharing)

### GraalVM version (if different from Java)

21.2.0

### Quarkus version or git rev

2.1.4.Final (also same failures on 1.13.2.FINAL)

### Build tool (ie. output of `mvnw --version` or `gradlew --version`)

Apache Maven 3.8.1 (05c21c65bdfed0f71a2f2ada8b84da59348c4c5d)

### Additional information

Also note, that this is being built in my CI pipeline using the image `quay.io/quarkus/centos-quarkus-maven:21.2-java11` (the environment fields were filled using that container's information. I should note that the image is able to build container images since the underlying CI builder node is using a docker agent to launch the container, so I believe JIB inside the container uses that to build the images (working theory at least). 

This is only a hunch, but I wonder if JIB is attempting to send an _authenticated_ pull (instead of anonymous) to docker hub, and the API doesn't like it, whereas the RedHat registry doesn't care? If this is the case, I would like to propose that if `quarkus.jib.base-registry-username` & `quarkus.jib.base-registry-password` are not set (which is currently default), then it should only attempt an _anonymous_ pull.

I have however been able to reproduce it on my local machine using the following condensed environment:
##### Local Environment
* Output of `uname -a` or `ver`: `Linux workstation 3.10.0-1127.18.2.el7.x86_64 #1 SMP Sun Jul 26 15:27:06 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux`
* Output of `java -version`: `openjdk version "11.0.11" 2021-04-20 LTS
OpenJDK Runtime Environment 18.9 (build 11.0.11+9-LTS)
OpenJDK 64-Bit Server VM 18.9 (build 11.0.11+9-LTS, mixed mode, sharing)`
* GraalVM version: `GraalVM 21.1.0 Java 11 CE` (Running under quay.io/quarkus/ubi-quarkus-native-image:21.1-java11 since I don't have GraalVM on my workstation)
* Quarkus Version: `2.1.4.FINAL`
* Build tool: `Apache Maven 3.8.1 (05c21c65bdfed0f71a2f2ada8b84da59348c4c5d)`

I wanted to make sure it wasn't something specific with my project, so I created a brand new project using the quarkus-maven-plugin (https://quarkus.io/guides/rest-json#creating-the-maven-project) and added JIB to the pom:
```
mvn io.quarkus.platform:quarkus-maven-plugin:2.1.4.Final:create \
    -DprojectGroupId=org.acme \
    -DprojectArtifactId=rest-json-quickstart \
    -DclassName="org.acme.rest.json.FruitResource" \
    -Dpath="/fruits" \
    -Dextensions="resteasy,resteasy-jackson"
cd rest-json-quickstart

mvn quarkus:add-extension -Dextensions="container-image-jib"

mvn clean package -Pnative -Dquarkus.container-image.build=true -Dquarkus.container-image.tag=0.0.1-SNAPSHOT-native  -Dquarkus.container-image.username=SAMPLE_USER -Dquarkus.container-image.password=SAMPLE_PASS -Dquarkus.container-image.registry=registry-url:5667 -Dquarkus.container-image.push=true -Dquarkus.jib.base-native-image=redhat/ubi8-minimal:8.4
```
This still produced the following stack trace for me :frowning_face: :
```
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  01:21 min
[INFO] Finished at: 2021-08-30T09:11:04-04:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal io.quarkus.platform:quarkus-maven-plugin:2.1.4.Final:build (default) on project rest-json-quickstart: Failed to build quarkus application: io.quarkus.builder.BuildException: Build failure: Build failed due to errors
[ERROR] 	[error]: Build step io.quarkus.container.image.jib.deployment.JibProcessor#buildFromNative threw an exception: java.lang.RuntimeException: Unable to create container image
[ERROR] 	at io.quarkus.container.image.jib.deployment.JibProcessor.containerize(JibProcessor.java:194)
[ERROR] 	at io.quarkus.container.image.jib.deployment.JibProcessor.buildFromNative(JibProcessor.java:169)
[ERROR] 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
[ERROR] 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
[ERROR] 	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
[ERROR] 	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
[ERROR] 	at io.quarkus.deployment.ExtensionLoader$2.execute(ExtensionLoader.java:820)
[ERROR] 	at io.quarkus.builder.BuildContext.run(BuildContext.java:277)
[ERROR] 	at org.jboss.threads.ContextHandler$1.runWith(ContextHandler.java:18)
[ERROR] 	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2449)
[ERROR] 	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1478)
[ERROR] 	at java.base/java.lang.Thread.run(Thread.java:829)
[ERROR] 	at org.jboss.threads.JBossThread.run(JBossThread.java:501)
[ERROR] Caused by: com.google.cloud.tools.jib.api.RegistryAuthenticationFailedException: Failed to authenticate with registry registry-1.docker.io/redhat/ubi8-minimal because: 401 Unauthorized
[ERROR] GET https://auth.docker.io/token?service=registry.docker.io&scope=repository:redhat/ubi8-minimal:pull
[ERROR] {"details":"incorrect username or password"}
[ERROR] 
[ERROR] 	at com.google.cloud.tools.jib.registry.RegistryAuthenticator.authenticate(RegistryAuthenticator.java:306)
[ERROR] 	at com.google.cloud.tools.jib.registry.RegistryAuthenticator.authenticate(RegistryAuthenticator.java:258)
[ERROR] 	at com.google.cloud.tools.jib.registry.RegistryAuthenticator.authenticatePull(RegistryAuthenticator.java:177)
[ERROR] 	at com.google.cloud.tools.jib.registry.RegistryClient.doBearerAuth(RegistryClient.java:336)
[ERROR] 	at com.google.cloud.tools.jib.registry.RegistryClient.authPullByWwwAuthenticate(RegistryClient.java:395)
[ERROR] 	at com.google.cloud.tools.jib.builder.steps.PullBaseImageStep.call(PullBaseImageStep.java:168)
[ERROR] 	at com.google.cloud.tools.jib.builder.steps.PullBaseImageStep.call(PullBaseImageStep.java:67)
[ERROR] 	at com.google.common.util.concurrent.TrustedListenableFutureTask$TrustedFutureInterruptibleTask.runInterruptibly(TrustedListenableFutureTask.java:125)
[ERROR] 	at com.google.common.util.concurrent.InterruptibleTask.run(InterruptibleTask.java:69)
[ERROR] 	at com.google.common.util.concurrent.TrustedListenableFutureTask.run(TrustedListenableFutureTask.java:78)
[ERROR] 	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
[ERROR] 	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
[ERROR] 	at java.base/java.lang.Thread.run(Thread.java:829)
[ERROR] Caused by: com.google.cloud.tools.jib.http.ResponseException: 401 Unauthorized
[ERROR] GET https://auth.docker.io/token?service=registry.docker.io&scope=repository:redhat/ubi8-minimal:pull
[ERROR] {"details":"incorrect username or password"}
[ERROR] 
[ERROR] 	at com.google.cloud.tools.jib.http.FailoverHttpClient.call(FailoverHttpClient.java:329)
[ERROR] 	at com.google.cloud.tools.jib.http.FailoverHttpClient.call(FailoverHttpClient.java:250)
[ERROR] 	at com.google.cloud.tools.jib.registry.RegistryAuthenticator.authenticate(RegistryAuthenticator.java:284)
[ERROR] 	... 12 more
[ERROR] Caused by: com.google.api.client.http.HttpResponseException: 401 Unauthorized
[ERROR] GET https://auth.docker.io/token?service=registry.docker.io&scope=repository:redhat/ubi8-minimal:pull
[ERROR] {"details":"incorrect username or password"}
[ERROR] 
[ERROR] 	at com.google.api.client.http.HttpRequest.execute(HttpRequest.java:1113)
[ERROR] 	at com.google.cloud.tools.jib.http.FailoverHttpClient.call(FailoverHttpClient.java:323)
[ERROR] 	... 14 more
[ERROR] -> [Help 1]
```
I think the key component is adding the  `-Dquarkus.container-image.username=SAMPLE_USER` and `-Dquarkus.container-image.password=SAMPLE_PASS` flags. Removing these flags actually produced a successful build and pushed the image to my private registry. Note that the push only worked because I have previously executed `docker login` on my local machine. In our CI pipeline, we are building in fresh containers each time, so passing in the username & password is required to authenticate with the private registry.

Sample output from running:
`mvn clean package -Pnative -Dquarkus.container-image.build=true -Dquarkus.container-image.tag=0.0.1-SNAPSHOT-native -Dquarkus.container-image.registry=private.registry:5667 -Dquarkus.container-image.push=true -Dquarkus.jib.base-native-image=redhat/ubi8-minimal:8.4`
```
[INFO] [io.quarkus.deployment.pkg.steps.NativeImageBuildRunner] docker run --env LANG=C --rm --user 1561400011:1561400011 -v /usr/local/data/git/TEST/rest-json-quickstart/target/rest-json-quickstart-1.0.0-SNAPSHOT-native-image-source-jar:/project:z --entrypoint /bin/bash quay.io/quarkus/ubi-quarkus-native-image:21.1-java11 -c objcopy --strip-debug rest-json-quickstart-1.0.0-SNAPSHOT-runner
[INFO] [io.quarkus.container.image.jib.deployment.JibProcessor] Starting container image build
[WARNING] [io.quarkus.container.image.jib.deployment.JibProcessor] Base image 'redhat/ubi8-minimal:8.4' does not use a specific image digest - build may not be reproducible
[INFO] [io.quarkus.container.image.jib.deployment.JibProcessor] LogEvent [level=INFO, message=Docker config auths section defines credentials for nexus.fl.udev.bits:5667]
[INFO] [io.quarkus.container.image.jib.deployment.JibProcessor] LogEvent [level=LIFECYCLE, message=Using credentials from Docker config (/home/cflores/.docker/config.json) for private.registry:5667/cflores/rest-json-quickstart:0.0.1-SNAPSHOT-native]
[INFO] [io.quarkus.container.image.jib.deployment.JibProcessor] The base image requires auth. Trying again for redhat/ubi8-minimal:8.4...
[INFO] [io.quarkus.container.image.jib.deployment.JibProcessor] Using base image with digest: sha256:31ccb79b1b2c2d6eff1bee0db23d5b8ab598eafd6238417d9813f1346f717c11
[INFO] [io.quarkus.container.image.jib.deployment.JibProcessor] Container entrypoint set to [./application]
[INFO] [io.quarkus.container.image.jib.deployment.JibProcessor] Pushed container image private-registry:5667/cflores/rest-json-quickstart:0.0.1-SNAPSHOT-native (sha256:2eb60522a62ca242bc20bd0014b4faeef3352c4cf4f40cffe7f02c1e8656d686)

[INFO] [io.quarkus.deployment.QuarkusAugmentor] Quarkus augmentation completed in 85958ms
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  01:34 min
[INFO] Finished at: 2021-08-30T09:23:10-04:00
[INFO] ------------------------------------------------------------------------
```
Is the `container-image-jib` plugin for some reason sending my private registry credentials to `docker.io`?

--- a/extensions/container-image/container-image-jib/deployment/src/main/java/io/quarkus/container/image/jib/deployment/JibProcessor.java
+++ b/extensions/container-image/container-image-jib/deployment/src/main/java/io/quarkus/container/image/jib/deployment/JibProcessor.java
@@ -161,7 +161,7 @@ public class JibProcessor {
                     "The native binary produced by the build is not a Linux binary and therefore cannot be used in a Linux container image. Consider adding \"quarkus.native.container-build=true\" to your configuration");
         }
 
-        JibContainerBuilder jibContainerBuilder = createContainerBuilderFromNative(containerImageConfig, jibConfig,
+        JibContainerBuilder jibContainerBuilder = createContainerBuilderFromNative(jibConfig,
                 nativeImage, containerImageLabels);
         setUser(jibConfig, jibContainerBuilder);
         setPlatforms(jibConfig, jibContainerBuilder);
@@ -517,7 +517,7 @@ public class JibProcessor {
         }
     }
 
-    private JibContainerBuilder createContainerBuilderFromNative(ContainerImageConfig containerImageConfig, JibConfig jibConfig,
+    private JibContainerBuilder createContainerBuilderFromNative(JibConfig jibConfig,
             NativeImageBuildItem nativeImageBuildItem, List<ContainerImageLabelBuildItem> containerImageLabels) {
 
         List<String> entrypoint;
@@ -532,8 +532,8 @@ public class JibProcessor {
         try {
             AbsoluteUnixPath workDirInContainer = AbsoluteUnixPath.get("/work");
             JibContainerBuilder jibContainerBuilder = Jib
-                    .from(toRegistryImage(ImageReference.parse(jibConfig.baseNativeImage), containerImageConfig.username,
-                            containerImageConfig.password))
+                    .from(toRegistryImage(ImageReference.parse(jibConfig.baseNativeImage), jibConfig.baseRegistryUsername,
+                            jibConfig.baseRegistryPassword))
                     .addFileEntriesLayer(FileEntriesLayer.builder()
                             .addEntry(nativeImageBuildItem.getPath(), workDirInContainer.resolve(BINARY_NAME_IN_CONTAINER),
                                     FilePermissions.fromOctalString("775"))

